contracts/src/test/kotlin/com/r3/corda/lib/tokens/contracts/TokenPointerTests.kt

```diff
@@ -49,7 +49,7 @@ class TokenPointerTests : ContractTestCommon() {
         // Check we can resolve the pointer inside the ledger transaction.
         val ledgerTransaction = testTransaction.toLedgerTransaction(aliceServices)
         val fungibleToken = ledgerTransaction.singleOutput<FungibleToken>()
-        assertEquals(((fungibleToken.tokenType) as TokenPointer<TestEvolvableTokenType>).pointer.resolve(ledgerTransaction), outputStateAndRef)
+        assertEquals(((fungibleToken.tokenType) as TokenPointer<*>).pointer.resolve(ledgerTransaction), outputStateAndRef)
```

Comment:
Wondering why you need to do this.

Reply:
There is compilation warning : Unchecked cast: TokenType to TokenPointer<TestEvolvableTokenType>
To resolve this had change that to <*>

Reply:
In this case I'd suggest add a @Suppress("UNCHECKED_CAST")
Rather then changing the expected type.